.github/workflows/tests.yml

```diff
@@ -82,10 +92,18 @@ jobs:
           python-version: ${{ matrix.pyversion }}
   
       - name: Set up Julia
+        id: setup_julia
+        if: ${{ matrix.juliaexe == 'julia' }}
```

Comment:
Please do this unconditionally as before.

Reply:
Fixed in ac95036. But why do we want to install it unconditionally?